addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -78,6 +79,15 @@ export class DateTimeField extends Component {
         return ensureArray(this.state.value);
     }
 
+    get placeholder() {
+        const dynamicPlaceholder =
+            this.props.placeholderField && this.props.record.data[this.props.placeholderField];
+        if (dynamicPlaceholder) {
+            return this.getFormattedValue(dynamicPlaceholder);
```

Comment:
Sorry to interject and intrude into the web scope. But I did a quick functional test to check the enterprise PR.
```suggestion
            return this.formatValue(dynamicPlaceholder);
```

Reply:
Indeed, thanks!